src/components/configurable-columns/README.md

```diff
@@ -0,0 +1,54 @@
+## Configurable Columns.
+
+This component and the `ConfigurableColumnRow` are building blocks for the `ConfigurableColumnsPattern`
+component that wires the two together.
+
+### How to use the Configurable Columns component:
+
+* In your file:
+
+```javascript
+import { ConfigurableColumns, ConfigurableColumnRow } from 'carbon/lib/components/configurable-columns';
+```
+
+* To render the Columns:
+
+```javascript
+
+rows(data) {
+  return (
+    <ol className='carbon-configurable-columns__columns-wrapper'>
```

Comment:
do they really need to define the `ol`? can that not just be a part of the parent component?